timvt/layer.py

```diff
@@ -129,23 +133,22 @@ async def get_tile(
                                 :xmin,
                                 :ymin,
                                 :xmax,
-                                :ymax,
-                                :epsg
+                                :ymax
                             ),
                             :seg_size
                         ) AS geom
                 ),
                 mvtgeom AS (
                     SELECT ST_AsMVTGeom(
-                        ST_Transform(t.{geometry_column}, :epsg),
+                        ST_Transform(t.{geometry_column}, {out_proj}),
                         bounds.geom,
                         :tile_resolution,
                         :tile_buffer
                     ) AS geom, {colstring}
                     FROM {self.id} t, bounds
                     WHERE ST_Intersects(
                         ST_Transform(t.{geometry_column}, 4326),
-                        ST_Transform(bounds.geom, 4326)
+                        ST_Transform(bounds.geom, {tms_proj}, 4326)
```

Comment:
Here we use `geometry ST_Transform(geometry geom, text from_proj, integer to_srid);`  (note: we remove the SRID in the `ST_MakeEnvelope` call so we need to add the input projection here)